modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -1389,16 +1346,15 @@ EXPLAIN INSERT INTO `travel-sample`.inventory.airport (key UUID(), value airport
         }
       ]
     },
-    "text": "INSERT INTO `travel-sample`.inventory.airport (key UUID(), value airport)\n    SELECT airport FROM `travel-sample`\n      WHERE type = \"airport\" AND airportname = \"Heathrow\";"
+    "text": "INSERT INTO `travel-sample`.inventory.airport (key UUID(), value airport)\n    SELECT airport FROM `travel-sample`.inventory.airport\n      WHERE airportname = \"Heathrow\";"
   }
 ]
 ----
 
-The Query Engine first executes the `SELECT` statement and then uses the projection to insert into the `travel-sample` keyspace, performing the operations in the order listed:
+The Query Engine first executes the `SELECT` statement and then uses the projection to insert into the `travel-sample`.inventory.airport keyspace, performing the operations in the order listed:
 
-<1> An `IndexScan` to find the documents that satisfy `type="airport"`.
-<2> A subsequent `IndexScan` for `airportname="Heathrow"`.
-<3> An `IntersectScan` to obtain the documents that satisfy both conditions of Step 1 and Step 2.
-<4> A `Fetch` for the value on the document `_airport`.
-<5> An `Insert` of the value along with the auto-generated key into the `travel-sample` keyspace.
+<1> An `IndexScan` to search for documents using the `def_inventory_airport_airportname` index.
```

Comment:
Given the output is now a little bit different, since we're changing the query, I am not sure if I've explained each point correctly, happy to take some suggestions here! 😓